bio.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+	<link rel="stylesheet" type="text/css" href="styles.css">
+	<title>Bio | Chandler Morrison</title>
+	<link href="https://fonts.googleapis.com/css?family=Inconsolata|Montserrat|Oxygen" rel="stylesheet">
+<ul class="NavBar">
+	<li><a href="index.html">Home</a></li>
```

Comment:
Elements such as ul or li (Or any other visual HTML element) belong in the body tag of the document. You may have confused "head" with "header", which is a very easy mix-up!